tests/test_series.py

```diff
@@ -436,6 +436,17 @@ def test_types_groupby() -> None:
     s.groupby(s > 2)
 
 
+def test_types_groupby_methods() -> None:
+    s = pd.Series([4, 2, 1, 8], index=["a", "b", "a", "b"])
+    check(assert_type(s.groupby(level=0).sum(), pd.Series), pd.Series)
```

Comment:
You could also check the dtype: `check(assert_type(s.groupby(level=0).sum(), "pd.Series[int]"), pd.Series, int)`